webApp-plivo/public/js/customclient.js

```diff
@@ -500,7 +501,7 @@ function tokenGenFunc(){
 * This is will prevent the other end still listening for dead call
 */
 window.onbeforeunload = function () {
```

Comment:
what happens when user closes the webpage ?

Reply:
This is abrupt disconnect from the client then this has to be handled at server end.